rfcs/text/0000-create-file-elf.md

```diff
@@ -0,0 +1,306 @@
+# 0000: Create the ELF sub-field of the File fieldset
+
+- Stage: **0 (strawperson)**
+- Date: **TBD**
+
+Create the Executable Linkable Format (ELF) sub-field, of the `file` top-level fieldset. This document metadata can be used for malware research, as well as coding and other application development efforts.
+
+## Fields
+
+**Stage 0**
+
+This RFC is to create the ELF sub-field within the `file.` fieldset. This will include 25 sub-fields.
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| file.elf.creation_date | date | Extracted when possible from the file's metadata. Indicates when it was built or compiled. It can also be faked by malware creators. |
+| file.elf.exports.name | keyword | Name of exported symbol |
```

Comment:
see comment about `imports` -- I'm thinking you're probably going to want this to be some sort of structured array, so either `flattened` or `nested` for `file.elf.exports`

Reply:
Yep, agreed. Fix't the oversight 👍